### PR TITLE
feat: add `last seen` column in user management table

### PR DESCRIPTION
**Pull request by keita-determined**
## Description
[WEB-1712]

Frontend change is to add `Last Seen` columns and support sort for all columns

Backend change is to make `last_log` optional and add sort for `last_login`. `last_login` should be renamed as `last_auth_at` or something similar but not in this PR

## Test Plan
- Check if `Last Seen` (last authenticated time) column is added in user management table
  - When `Last Seen` is not available, `N/A` is on display
- Check if each column is sortable by clicking header

## Checklist

- [x] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket

[WEB-1712]: https://hpe-aiatscale.atlassian.net/browse/WEB-1712?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: proto/src/determined/user/v1/user.proto [hunk 1]
```diff
@@ -28,8 +28,8 @@ message User {
   // Bool denoting whether the user should be able to login with or change a
   // password.
   bool remote = 8;
-  // when the user last logged in
-  google.protobuf.Timestamp last_login = 9;
+  // when the user last authenticated
+  optional google.protobuf.Timestamp last_login = 9;
```
Comment (keita-determined) [inline proto/src/determined/user/v1/user.proto:32]: I think `last_auth` is more accurate than `last_login`, but leave it as it is for now

**Review by ashtonG — APPROVED** [review]
frontend looks good

File: proto/src/determined/user/v1/user.proto (hunk 1 above)
Comment (NicholasBlaskey) [reply proto/src/determined/user/v1/user.proto:32]: This hasn't been released yet right? 

I would try and avoid a backwards incompatible change to ```GetUsers``` from release to release?

**Review by NicholasBlaskey — APPROVED** [review]
LGTM

File: proto/src/determined/user/v1/user.proto (hunk 1 above)
Comment (keita-determined) [reply proto/src/determined/user/v1/user.proto:32]: Not released yet.

**Pull request merged**